shell/platform/windows/system_utils_unittests.cc

```diff
@@ -4,28 +4,67 @@
 
 #include <cwchar>
 
+#include "flutter/fml/platform/win/wstring_conversion.h"
 #include "flutter/shell/platform/windows/system_utils.h"
 #include "gtest/gtest.h"
 
 namespace flutter {
 namespace testing {
 
+class MockWindowsRegistry : public WindowsRegistry {
+ public:
+  virtual ~MockWindowsRegistry() {}
+
+  virtual LSTATUS GetRegistryValue(HKEY hkey,
+                                   LPCWSTR key,
+                                   LPCWSTR value,
+                                   DWORD flags,
+                                   LPDWORD type,
+                                   PVOID data,
+                                   LPDWORD sizeData) const {
+    static const wchar_t* locales =
+        L"en-US\0zh-Hans-CN\0ja\0zh-Hant-TW\0he\0\0";
+    static DWORD locales_len = 35;
```

Comment:
Is there any way to get the length of `locales` programmatically?
> https://stackoverflow.com/questions/2853615/get-length-of-wchar-t-in-c

Reply:
The issue here is that it's not just a string, it's a multistring, which means it contains null delimiters in the middle that string length methods will treat as a terminator and return a bad value for. We could write our own method that counts until it reaches a double null, though I don't know how much use it would get outside of this test.

Reply:
Is the incoming data a string of locales delimited by `\0`?

Reply:
Yes

Reply:
Lol, I wonder why they would do that 🤷‍♂️ . Anyway, the change LGTM now 👍 .

Reply:
You _should_ be able to do this as follows:
```
using namespace std::string_literals;
static const std::wstring locales = L"en-US\0zh-Hans-CN\0ja\0zh-Hant-TW\0he\0\0"s;
static const DWORD locales_len = locales.size();
```

Reply:
That said -- the documentation for the `pcbData` param to `GetRegValue` function states this value should be in bytes, so we should really return:
```cc
static const DWORD locales_len = locales.size() * sizeof(wchar_t);
```
which in this case works out to 140.